projects/chaos/src/main.c

```diff
@@ -90,6 +88,9 @@ int main(void) {
   emergency_fault_clear(&s_emergency_storage);
   state_handler_init();
 
+  // GPIO
+  gpio_fsm_init(cfg);
```

Comment:
A comment explaining why this ordering is important would be nice.

Reply:
Done.